bigtable/google/cloud/bigtable/row_set.py

```diff
@@ -138,6 +133,24 @@ def __init__(self, start_key=None, end_key=None,
         self.end_key = end_key
         self.end_inclusive = end_inclusive
 
+    def _key(self):
+        """A tuple key that uniquely describes this field.
+
+        Used to compute this instance's hashcode and evaluate equality.
+
+        Returns:
+            Tuple[str]: The contents of this :class:`.RowRange`.
+        """
+        return (
+            self.start_key,
+            self.start_inclusive,
+            self.end_key,
+            self.end_inclusive,
+        )
```

Comment:
Would it make sense to use `_key` below in `_eq`, as well?

Reply:
Yes. Updated the code, thanks.